src/main/java/commands/roles/AddRoleAssign.java

```diff
@@ -33,29 +34,25 @@ public CommandState canBeExecuted(long guildId, long channelId, Member member) {
     public void run(String[] args, GuildMessageReceivedEvent e) throws Exception {
         DataHandler dataHandler = new DataHandler();
         long guildId = e.getGuild().getIdLong();
-        if (args.length == 1) {
-            e.getChannel().sendMessage("No emoji, role and name given\n" + getUsage()).queue();
-        } else if (args.length == 2) {
-            e.getChannel().sendMessage("No role and name given\n" + getUsage()).queue();
-        } else if (args.length == 3) {
-            e.getChannel().sendMessage("No name given\n" + getUsage()).queue();
-        } else if (args.length >= 4 && dataHandler.getRoleCategories(e.getGuild().getIdLong()).contains(args[0])) {
-            if (!hasEmoji(e.getMessage(), args[1])) {
-                e.getChannel().sendMessage(String.format("%s is not a valid emoji\n%s", args[1], getUsage())).queue();
-                return;
-            }
+        if (args.length == 1)
```

Comment:
I believe that a switch would be more readable here.

Reply:
Will fix together with #76